extensions/amp-story-shopping/0.1/amp-story-shopping-tag.css

```diff
@@ -13,6 +13,15 @@ amp-story-shopping-tag {
   --i-amphtml-shopping-tag-backdrop-filter: blur(20px);
 }
 
+
+.amp-story-shopping-tag-currency-symbol {
```

Comment:
We can probably remove `font-family` for this rule since it's already specified with the same value in `amp-story-shopping-tag`

Reply:
yes, done.